openbas-api/src/main/java/io/openbas/migration/V3_54__Add_obfuscator_inject_contract.java

```diff
@@ -0,0 +1,53 @@
+package io.openbas.migration;
+
+import java.sql.Connection;
+import java.sql.Statement;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+@Component
+public class V3_54__Add_obfuscator_inject_contract extends BaseJavaMigration {
```

Comment:
**question**
Can you try to just relaunch your app and see if you really need this ?
I suppose, the contract are updated when the app relaunch.

Reply:
I try if i don't have this script, when i relaunch it does not update the existing payloads... so we need it